src/client/components/YesNoQuestion/index.js

```diff
@@ -35,7 +37,10 @@ function YesNoQuestion(props) {
 
   return (
     <div className="bg-light p-2 m-2">
-      <Form.Group controlId={inputName}>
+      <Form.Group
+        controlId={inputName}
+        className={isYes === undefined ? "unchecked" : ""}
```

Comment:
This should be added to Question 7's followup as well. The DisasterQuestion component. @jlechowicz is also making updates to that file (it's changing to radio buttons), so you might want to coordinate with him.

Reply:
Yeah, I'll wait until he changes to a radio button then we can add the feedback. With a dropdown, there's no need for feedback because something is always selected.